src/views/User/MyPostPage/MyPostPage.jsx

```diff
@@ -1,7 +1,57 @@
 import React from 'react';
+import styled from '@emotion/styled';
+import { ShortHeader } from '@/components/Header';
+import { PostCard } from '@/components/PostCard';
+import { postsData } from '@/assets/data.js';
 
 const MyPostPage = () => {
-  return <div></div>;
+  const { posts } = postsData;
+  const postLength = 0;
```

Comment:
```suggestion
  const postLength = posts.length;
```

테스트용인 것 같은데 서버 데이터를 가져와도 위와 같은 형태가 될 것 같아요